src/main/java/step2/StringCalculator.java

```diff
@@ -0,0 +1,21 @@
+package step2;
+
+public class StringCalculator {
+
+    public int calculate(Factor factor, int result) {
+        Number first = factor.getNumbers(true, result);
+        Number second = factor.getNumbers(false, result);
+
+        Operator operator = factor.getOperator();
+
+        result = calculate(first, operator, second);
+
+        if(factor.isComplete()) return result;
```

Comment:
```suggestion
        if(factor.isComplete()) {
            return result;
         }
```
자바 컨벤션에 따라 if문 한줄이라도 {}를 쓰는게 좋습니다 :)